api/envoy/api/v2/cds.proto

```diff
@@ -157,6 +158,21 @@ message Cluster {
   // then hosts is required.
   repeated core.Address hosts = 7;
 
+  // Setting this is required for specifying members of
+  // :ref:`STATIC<envoy_api_enum_value_Cluster.DiscoveryType.STATIC>`,
+  // :ref:`STRICT_DNS<envoy_api_enum_value_Cluster.DiscoveryType.STRICT_DNS>`
+  // or :ref:`LOGICAL_DNS<envoy_api_enum_value_Cluster.DiscoveryType.LOGICAL_DNS>` clusters.
+  // This field supersedes :ref:`hosts<envoy_api_field_Cluster.hosts>` field.
+  //
+  // .. attention::
+  //
+  //   Setting this allows CDS static/DNS assignments to contain embedded EDS equivalent
+  //   :ref:`endpoint assignments<envoy_api_msg_ClusterLoadAssignment>`.
+  //   Setting this overrides :ref:`hosts<envoy_api_field_Cluster.hosts>` values.
+  //
+  // [#not-implemented-hide:]
+  ClusterLoadAssignment load_assignment = 33;
```

Comment:
I think the discussion at https://github.com/envoyproxy/envoy/pull/3261/files#r188243657 still applies.

Reply:
@zuercher thanks, will update it.

Reply:
Updated in 0001298.